packages/php-wasm/compile/php/php_wasm.c

```diff
@@ -396,7 +396,7 @@ PHP_FUNCTION(post_message_to_js)
 
 	char *response;
 	size_t response_len = js_module_onMessage(data, &response);
-	if (response != -1)
+	if (response_len != -1)
```

Comment:
JS returns -1 if there was an error processing the message to `response_len`. Checking `response` was the error.